lib/graphql/schema.flow.js

```diff
@@ -14,7 +14,7 @@ export type Scalars = {|
 
 /** The bank account of the current user */
 export type Account = {|
-  __typename?: 'Account',
+   __typename?: 'Account',
```

Comment:
Do I have some incorrect setting that formatted this with an extra space??

Reply:
I had the same yesterday. Decided to include my schema changes :)

Reply:
I also have the same locally, the change was introduced by Sergey when he ran the command previously, see: https://github.com/kontist/js-sdk/pull/100#discussion_r432493232

It might have to do with environment settings or something, I'd rather keep the version that most of us have, so I'm fine with keeping these extra changes